### PR TITLE
Start / Stop Geth Node in Livepeer

### PR DESCRIPTION
**Pull request by ericxtang**

### CODE REVIEW

File: cmd/livepeer/livepeer.go [hunk 1]
```diff
@@ -181,8 +175,34 @@ func main() {
...
+	var gethCmd *exec.Cmd
 	//Set up ethereum-related stuff
-	if *gethipc != "" {
+	if *ethDatadir != "" {
+		gethipc := filepath.Join(filepath.Join(*ethDatadir, "geth.ipc"))
+		//If getipc file is not there, start the geth node
+		if _, err := os.Stat(gethipc); os.IsNotExist(err) {
+			//Set up geth params depending on network, invoke geth
+			if *testnet {
+				gethCmd = exec.Command("geth", "--rpc", "--datadir", *ethDatadir, "--bootnodes=enode://a1bd18a737acef008f94857654cfb2470124d1dc826b6248cea0331a7ca82b36d2389566e3aa0a1bc9a5c3c34a61f47601a6cff5279d829fcc60cb632ee88bad@13.58.149.151:30303") //timeout in 3 mins
```
Comment (dob) [inline cmd/livepeer/livepeer.go:186]: so here we hard code the geth bootnode, but above we have a flag for the livepeer bootnode? I could see this getting a little confusing.

File: cmd/livepeer/livepeer.go [hunk 2]
```diff
@@ -219,7 +252,7 @@ func main() {
 		n.EthPassword = *ethPassword
 
 		if *deposit > 0 {
```
Comment (dob) [inline cmd/livepeer/livepeer.go:254]: What's the point of the deposit flag? Just a timesaver? I feel like we could keep starting the node separate from depositing, and instead just use the cli to deposit.

**Review by dob — APPROVED** [review]
There are some media server changes in here to. Fine if intentional, but just wanted to point them out in case they snuck in by accident. I approve this :ship:, but may want to address the comments first anyway.

File: cmd/livepeer/livepeer.go (hunk 2 above)
Comment (yondonfu) [reply cmd/livepeer/livepeer.go:254]: So the deposit flag was added before the cli wizard was created to allow a broadcaster to optionally deposit tokens, but now that the cli wizard can deposit tokens, we shouldn't need this flag anymore

Comment (ericxtang) [reply cmd/livepeer/livepeer.go:254]: Created https://github.com/livepeer/go-livepeer/issues/95 to address this

File: cmd/livepeer/livepeer.go (hunk 1 above)
Comment (ericxtang) [reply cmd/livepeer/livepeer.go:186]: Ahh... the eth bootnode is different from the Livepeer bootnode.  Here, we are hard coding to our private eth testnet bootnode.  The flags specify Livepeer bootnodes.  (The user shouldn't have to worry about eth bootnode)

**Comment by ericxtang** [comment]
@dob I snuck in a bug fix in mediaServer here.  It's fixing some video playback issues for transcoded video.

**Pull request merged**